config/schedule.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+set :output, "/tmp/figgy_update_bib_ids.log"
+set :job_template, "bash -l -c 'export PATH=\"/usr/local/bin/:$PATH\" && :job'"
+job_type :logging_rake, "cd :path && :environment_variable=:environment bundle exec rake :task :output"
+every :day, at: "2:00 am", roles: [:db] do
```

Comment:
Roles are pulled from Capistrano so this job is only run on figgy1?

Reply:
Also, does this time conflict with any Orangelight indexing operations? Or sitemap regeneration?

Reply:
`db` is only for `figgy1` on the production environment: https://github.com/pulibrary/figgy/blob/master/config/deploy/production.rb#L10

Reply:
Role based execution is a huge advantage of whenever over a cron job managed in Ansible. As to the other issues, OL sitemap regen occurs at 2AM every night; other indexing tasks don't seem to be in conflict. Not sure if this is an issue, it's just that Solr might be taxed as these other jobs run at the same time.